docs/gateway-api/gateway-api-v1beta1.md

```diff
@@ -0,0 +1,247 @@
+
+A sample status of the Gateway is shown below:
+
+  ```yaml
+  status:
+    addresses:
+    - type: IPAddress
+      value: 10.1.1.12
+    conditions:
+    - lastTransitionTime: "2023-08-31T15:04:14Z"
+      message: Gateway configuration is valid
+      observedGeneration: 1
+      reason: Accepted
+      status: "True"
+      type: Accepted
+    - lastTransitionTime: "2023-08-31T15:04:16Z"
+      message: Virtual service configured/updated
+      observedGeneration: 1
+      reason: Programmed
+      status: "True"
+      type: Programmed
+    listeners:
+    - attachedRoutes: 1
+      conditions:
+      - lastTransitionTime: "2023-08-31T15:07:35Z"
+        message: Listener is valid
+        observedGeneration: 1
+        reason: Accepted
+        status: "True"
+        type: Accepted
+      name: http
+      supportedKinds:
+      - group: gateway.networking.k8s.io
+        kind: HTTPRoute
+  ```
+
+A sample HTTPRoute status is shown below:
+
+  ```yaml
+  status:
+    parents:
+    - conditions:
+      - lastTransitionTime: "2023-09-06T11:49:06Z"
+        message: Parent reference is valid
+        observedGeneration: 1
+        reason: Accepted
+        status: "True"
+        type: Accepted
+      controllerName: ako.vmware.com/avi-lb
+      parentRef:
+        group: gateway.networking.k8s.io
+        kind: Gateway
+        name: my-gateway
+        namespace: default
+        sectionName: http
+  ```
+
+### Conditions and Caveats
+
+#### Gateway Limitations
+
+AKO accepts the following Gateway configuration for this release:
+  
+  1. Gateway MUST contain at least one listener configuration in it.
+  2. Gateway MUST NOT contain protocols other than HTTP or HTTPS.
+  3. Gateway MUST contain a hostname. Hostname as `*` is not supported and `*.domain` is supported.
+  4. Gateway MUST NOT contain TLS modes other than `Terminate`.
+
+#### HTTPRoute Limitations
+
+AKO accepts the following HTTPRoute configuration for this release:
+
+  1. HTTPRoute MUST contain at least one parent reference.
+  2. HTTPRoute MUST NOT contain `*` as hostname.
+  3. HTTPRoute MUST contain at least one hostname match with parent Gateway
+
+#### Resource Creation
+
+For the Tech preview, AKO imposes a restriction on the order of GatewayAPI object creation. An object that is referenced must be created first. For example, GatewayClass must be created before Gateway and Gateway before HTTPRoute creation. This restriction is only applicable to the Gateway API objects and will be removed in the future releases.
+
+#### High Availability Support
+
+For the Tech Preview, High Availability is not supported.
+
+#### Configuring Static IP address
+
+The AKO supports gateway objects with a single IPv4 address. The user can their preferred static IPv4 address by specifying `spec.addresses` in the Gateway object. A sample configuration is shown below:
```

Comment:
- "The user can their" should be changed to "The user can specify their".
-  Might want to capitalise first letter in 'gateway' (proper noun)

Also , we haven't mentioned anything about child vs, Pool and PoolGroup naming conventions anywhere. Should we include that too ?

Reply:
The pool, pg, and all are added above while explaining the httproute to child vs modeling. Pool and PG naming is one thing I would like to add. but that will change in the future.